core/tests/karma.conf.ts

```diff
@@ -150,7 +150,13 @@ module.exports = function(config) {
           'node_modules',
           'third_party',
         ],
-        extensions: ['.ts', '.js', '.json', '.html', '.svg', '.png']
+        extensions: ['.ts', '.js', '.json', '.html', '.svg', '.png'],
+        alias: {
+          // This is needed because in app.constans.ts we need to import
```

Comment:
constants is misspelled. Please fix here and everywhere else.

Reply:
done